pkgs/development/tools/analysis/flow/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, fetchFromGitHub, lib, ocaml, libelf, cf-private, CoreServices, findlib, camlp4, sedlex, ocamlbuild }:
```

Comment:
Does it need specifically lwt 2? It seems to work well with more recent versions of lwt; probably you should use the generic `ocaml_lwt` attribute.

Reply:
I tried with `ocaml_lwt` and it worked fine too. I'm committing.